pasta_eln/GUI/project.py

```diff
@@ -254,7 +254,8 @@ def execute(self, command:list[Any]) -> None:
         self.comm.changeSidebar.emit('redraw')
         self.comm.changeTable.emit('x0','')
     elif command[0] is Command.SCAN:
-      self.comm.backend.scanProject(self.comm.progressBar, self.projID, self.docProj['-branch'][0]['path'])
+      for _ in range(2):  #scan twice: convert, extract
```

Comment:
Why two times always? What is the necessity for doing this twice always?

Reply:
Scan project could potentially (I build useful Use cases) create a new file, and this new file would have to be included in the second time doing the scan. Existing files are not modified the second time, so ...
- scan 1: scan for new files and potentially create a new file
- scan 2: scan for new files. I don't see a use case where these create new files.

Reply:
@SteffenBrinckmann That means scan creates files which needs to be re-scanned? How is it sure that the second scan will not generate any additional results which needs to be re-scanned too?

Reply:
@jmurugan-fzj The second scan can generate additional files, and since that is in the users choice (the user can change those add-ons=extractors), one cannot be sure.
- The one thing that could make it sure, is to rescan every say 5min. 
- It is similar to git: you can do a command / press a button to see if there are changes on the server that you should pull (that is how it is currently implemented in Pasta, to manually execute the "scan for new"); or one checks every ... min for changes and (1) auto-pulls them OR (2) inform the user that he should pull them [like many IDEs do].

Coming back to the original question: I don't see why the user would create a file in the 2nd scan.

Reply:
@SteffenBrinckmann Okay fine, so it's user's responsibility to do the scan manually and import any generated results, will there be possibilities that the user may not be aware of such generated results?